docs/commands/rpc-protocol.md

```diff
@@ -341,6 +341,27 @@ Returns how many RAW is owned and how many have not yet been received by **accou
 }
 ```  
 
+!!! info "Error handling"
+    With _version 24.0+_, `accounts_balances` response errors are also returned per entry.
+    ```json
+    {
+      "balances": {
+        "nano_3wfddg7a1paogrcwi3yhwnaerboukbr7rs3z3ino5toyq3yyhimo6f6egij6": {
+          "balance": "442000000000000000000000000000",
+          "pending": "0",
+          "receivable": "0"
+        },
+        "nano_36uccgpjzhjsdbj44wm1y5hyz8gefx3wjpp1jircxt84nopxkxti5bzq1rnz": {
+          "error": "Bad account number"
+        },
+        "nano_1hrts7hcoozxccnffoq9hqhngnn9jz783usapejm57ejtqcyz9dpso1bibuy": {
+          "error": "Account not found"
```

Comment:
accounts_balances will never return "Account not found".
It will simply return zero balances when account does not exist.

Reply:
Have you tested it?

Reply:
Let's documents is like this:
* v24.0 is broken for unopened accounts with receivables
* v24.1 and greater will never return Account not found

Reply:
It is not clear to me what is broken in V24.0 and that V24.1 fixes. Could you send a commit fixing it @dsiganos ?

Reply:
@thsfs done, please check